### PR TITLE
edited documentation

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
@gwaybio Made some small edits to the repo, mainly documentation.

### CODE REVIEW

File: 1.segment-data/segment-cell-health-data.ipynb
Comment (gwaybio) [inline 1.segment-data/segment-cell-health-data.ipynb]: Can you make sure that you're using the correct conda kernel?

File: 2.extract-features/DP-project-utils.py
Comment (gwaybio) [inline 2.extract-features/DP-project-utils.py]: 😱

File: 2.extract-features/README.md
Comment (gwaybio) [inline 2.extract-features/README.md:13]: nice catch

File: 2.extract-features/README.md
Comment (gwaybio) [inline 2.extract-features/README.md:75]: 🕵️‍♀️

File: 2.extract-features/compile-DP-projects.ipynb
Comment (gwaybio) [inline 2.extract-features/compile-DP-projects.ipynb:62361]: I feel like this kernel is wrong too (it should be a conda env)

**Review by gwaybio — CHANGES_REQUESTED** [review]
Nice catches on these items.

Please correct the conda kernels then this should be good to go.

File: 1.segment-data/segment-cell-health-data.ipynb
Comment (jenna-tomkinson) [reply 1.segment-data/segment-cell-health-data.ipynb]: Woahhh interesting! I didn't know that the kernel shows. Ill add the environment on my end and change the kernel. I just opened it to check, good to know opening it in a different kernel makes an impact.

**Comment by jenna-tomkinson** [comment]
@gwaybio The kernels should be fixed. FYI I don't have permissions to merge PR requests on this machine (as far as I can see).

**Review by gwaybio — CHANGES_REQUESTED** [review]
- Please fix red symbol: https://github.com/WayScience/cell-health-data/pull/9/files#r969804061
- Please nbconvert `.ipynb` files (if not converted already)

Also note that we require one approval prior to merging :)

**Comment by jenna-tomkinson** [comment]
@gwaybio Error should be fixed (turned on "insert final newline" setting) and a new .py file made to reflect the added documentation.

**Comment by gwaybio** [comment]
I still see the red symbol 😱 https://github.com/WayScience/cell-health-data/pull/9/files

There are four empty spaces in the new line at the end, and no new line after the four empty spaces. Maybe you forgot to save the file and add after adjusting `vscode`?

After you fix this, I will approve and merge 💯

**Comment by jenna-tomkinson** [comment]
@gwaybio The issue is fixed. Looks like when I added the line manually, it was automatically indented so Github didn't see it as a new line.

**Review by gwaybio — APPROVED** [review]
Looks great! 💯

**Pull request merged**
